packages/tools/devtools/devtools-browser-extension/tsconfig.json

```diff
@@ -1,9 +1,8 @@
 {
-	"extends": "@fluidframework/build-common/ts-common-config.json",
+	"extends": "../../../../common/build/build-common/tsconfig.node16.json",
 	"compilerOptions": {
 		"rootDir": "src",
-		"outDir": "dist/tsc",
-		"composite": true,
+		"outDir": "./lib",
```

Comment:
maybe this should be `./lib/tsc`?

Reply:
Hmm...  I'll bet someone added this to avoid comingling the tsc output (which I don't think is actually used) and the webpack output.

I'll put it back.  Good catch.

Reply:
I don't recall exactly, but this might have had to do with a deliberate change we did because otherwise the tests-with-code-coverage run was analyzing _everything_ in the webpack bundle.